frontend/src/scenes/insights/InsightTabs/FunnelTab/FunnelConversionWindowFilter.tsx

```diff
@@ -74,14 +74,15 @@ export function FunnelConversionWindowFilter(): JSX.Element {
                 <Select
                     ref={timeUnitRef}
                     className="time-unit-input"
-                    defaultValue={FunnelConversionWindowTimeUnit.Day}
+                    defaultValue={conversionWindow.funnel_window_interval_unit}
                     dropdownMatchSelectWidth={false}
                     value={localConversionWindow.funnel_window_interval_unit}
                     onChange={(funnel_window_interval_unit: FunnelConversionWindowTimeUnit) => {
                         setLocalConversionWindow((state) => ({ ...state, funnel_window_interval_unit }))
-                        timeUnitRef.current?.blur()
+                        setFilters({
+                            funnel_window_interval_unit,
+                        })
                     }}
-                    onBlur={onChange}
```

Comment:
Sorry, what's the change here? It should trigger on change here now

Reply:
Hmm i'm trying it out and it feels very unintuitive. My first reaction after changing the parameter is "why isn't this chart updating" especially since all other filters change immediately on value change

Reply:
@alexkim205

Reply:
I was pointed at this by a customer report during support hero week

I've updated this to use an existing `use-debounce` dependency. So we can always call the set method and let the debounce smooth it for the user

We could simplify this further. But feels like right now, it's broken so (almost) anything is an improvement :)